test/e2e/kubectl/kubectl.go

```diff
@@ -801,6 +803,67 @@ metadata:
 			framework.ExpectNoError(c.CoreV1().Pods(ns).Delete(ctx, "run-test-3", metav1.DeleteOptions{}))
 		})
 
+		ginkgo.It("should support inline execution and attach with websockets", func(ctx context.Context) {
+			waitForStdinContent := func(pod, content string) string {
+				var logOutput string
+				err := wait.Poll(10*time.Second, 5*time.Minute, func() (bool, error) {
```

Comment:
the linter complains here https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/119186/pull-kubernetes-linter-hints/1700964135329599488

```
RROR: test/e2e/kubectl/kubectl.go:809:12: SA1019: wait.Poll is deprecated: This method does not return errors from context, use PollUntilContextTimeout. Note that the new method will no longer return ErrWaitTimeout and instead return errors defined by the context package. Will be removed in a future release. (staticcheck)
ERROR: 				err := wait.Poll(10*time.Second, 5*time.Minute, func() (bool, error) {
ERROR: 			
```

Reply:
the e2e test is working though , great

> Kubernetes e2e suite: [It] [sig-cli] Kubectl client Simple pod should support inline execution and attach with websockets

Reply:
Fixed linter issue.